project/gamestats_keys.txt

```diff
@@ -1,5 +1,5 @@
-# Gamestats HTTP keys
```

Comment:
Could you please remove this change as it seems unnecessary.

Reply:
Sorry, it was an accident when I was sorting the new keys. It's fixed now.

Reply:
No worries and thanks again for the keys ;v)